README.rst

```diff
@@ -196,3 +196,20 @@ This version of FogLAMP relies on SQLite to run. SQLite is embedded into the Sto
 where *user* is the name of the Linux user that will run FogLAMP. The FogLAMP database user must have *createdb* privileges (i.e. the *-d* argument).
 |br| |br|
 
+
+Troubleshooting
+===============
+
+FogLAMP version 1.7.0
+---------------------
+
+$FOGLAMP_ROOT/data/etc directory ownership
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The execution of the *sudo make install* immediately after *git clone* will create a *data/etc* directory owned by the *root* user,
+it should be owned by the user that will run FogLAMP, to fix it:
+::
+    chown -R <user>:<user> $FOGLAMP_ROOT/data
```

Comment:
May be this can simply be `chown -R $(whoami):$(whoami) $FOGLAMP_ROOT/data`

Reply:
I have followed the same sturture of the test above : 

```
If You Use PostgreSQL: Creating the Database Repository
=======================================================

This version of FogLAMP relies on SQLite to run. SQLite is embedded into the Storage service, but you may want to use PostgreSQL as a buffer and metadata storage (refer to the documentation on `ReadTheDocs <http://foglamp.readthedocs.io>`_ for more info. With a version of PostgreSQL installed via *apt-get* first you need to create a new database user with:
::
   sudo -u postgres createuser -d <user>

where *user* is the name of the Linux user that will run FogLAMP. The FogLAMP database user must have *createdb* privileges (i.e. the *-d* argument).
|br| |br|
```

Reply:
Okay. Consistency is a good thing to have. 👍

Reply:
done